ipatests/test_integration/test_sssd.py

```diff
@@ -318,7 +318,7 @@ def test_ext_grp_with_ldap(self):
         self.master.run_command(
             ['ipa', 'group-add-member', '--group', ext_group, user])
         self.master.run_command([
-            'ipa', 'group-add-member', '--external',
+            'ipa', '-n', 'group-add-member', '--external',
             self.users['ad']['name'], ext_group,
             '--users=', '--groups='])
```

Comment:
With the -n option I guess you can remove --users= --groups=

Reply:
Fixed, thanks.